src/Views/SearchView.vala

```diff
@@ -66,10 +66,21 @@ public class AppCenter.Views.SearchView : AbstractView {
     }
 
     public void search (string search_term, AppStream.Category? category, bool mimetype = false) {
+        /*
+            Annotation 01
+            Atheesh Thirumalairajan (@candiedoperation)
+            https://github.com/candiedoperation
+            
+            This Method has been implement in order to
+            update the dynamic FlatHub Link if no apps
+            match the search term.
+        */
+        
```

Comment:
If you want to add yourself to copyright, please do so in the copyright header. Otherwise, there's git blame for this sort of thing.